overlord/fdestate/backend/reseal.go

```diff
@@ -70,6 +70,17 @@ type SealingParameters struct {
 	TpmPCRProfile []byte
 }
 
+// EncryptedDisk gives information on the role, path and path to extra
+// legacy keys.
+type EncryptedDisk interface {
```

Comment:
disk vs partition? Maybe `EncryptedStore` ?

Reply:
not Store please

Reply:
indeed, another abused term :smile:

Reply:
EncryptedContainer to make it the same as other names.

Reply:
Done.